README.md

```diff
@@ -48,6 +48,12 @@ print(f"Valid webhook signature: {valid}")
 
 ## Run tests
 
+Before run integration tests:
+- test/test_get_account_statement.py
+
+please perform the steps:
+1. Get sandbox api key according [the docs here](https://api-docs.transferwise.com/#payouts-guide-api-access)
+2. Set an environment variable 'TRANSFERWISE_API_KEY' with a value of your sandbox api key. 
```

Comment:
These instructions are improving - do you mind if I provide some rephrasing suggestions?